.tool-versions

```diff
@@ -2,6 +2,7 @@
 
 actionlint 1.7.7
 cosign 2.5.0
+diffoci 0.1.6
```

Comment:
This dependency should be added to the [contributing guidelines](https://github.com/ericcornelissen/js-regex-security-scanner/blob/dbe4b993186f28cdbf35f6a4947744fbc5470797/CONTRIBUTING.md?plain=1#L87-L92) too.

Reply:
Resolved with 79925dbdb89e7fde887a9646cb2d2d0312e6106e